tests/graphql/passed/test-nullable-values-query.graphql

```diff
@@ -0,0 +1,19 @@
+#> server-sdl
+schema {
+  query: Query
+}
+
+type Query {
+  user(id: ID!): User @http(path: "/user/{{args.id}}", baseURL: "http://localhost:8080")
+}
+
+type User {
+  id: ID!
+  name: String!
+}
+
+type Post {
+  id: Int!
+  userId: Int
+  user: User @http(path: "/users", query: [{key: "id", value: "{{value.userId}}"}], baseURL: "http://localhost:8080")
+}
```

Comment:
We should add a `#> client-sdl` to make sure we are able to generate the correct schema.